server/src/main/java/org/elasticsearch/search/rescore/QueryRescorerBuilder.java

```diff
@@ -250,6 +250,7 @@ public QueryRescorerBuilder rewrite(QueryRewriteContext ctx) throws IOException
         queryRescoreBuilder.setQueryWeight(queryWeight);
         queryRescoreBuilder.setRescoreQueryWeight(rescoreQueryWeight);
         queryRescoreBuilder.setScoreMode(scoreMode);
+        queryRescoreBuilder.windowSize = this.windowSize
```

Comment:
This is missing a semicolon to end the line, so it doesn't compile.
Also, could you use the super class setter `windowSize(int windowSize)` instead of setting the field directly? In this case it doesn't matter much but sometimes we add input checks to the setters and this would circumvent potential validation checks.

Reply:
oops, will use the setter and a null check.
I used direct field access to avoid a NPE (it's stored as an Integer but the setting signature has int).

Reply:
> I used direct field access to avoid a NPE

ah, that makes sense.